tests/unit/__snapshots__/storybook.spec.ts.snap

```diff
@@ -143,15 +143,17 @@ exports[`Storyshots InteractionDesigner/Toolbar Default 1`] = `
                        
                       <!---->
                        
```

Comment:
@poojahpatil90 , I'm happy you quickly learnt how to handle this, good job

Reply:
Thanks!  My knowledge on storybook is still limited. I am looking forward to learning and playing around with it more.